src/_tutorials/dart-vm/httpserver.md

```diff
@@ -702,14 +702,14 @@ Future main() async {
    characters can be encoded over multiple bytes. The join() method puts the
    chunks together.
 
-3. The URL for the request is
+3. The data sent by the client is JSON formatted. The server decodes it using
+   the JSON codec available in the [dart:convert][] library.
+
+4. The URL for the request is
```

Comment:
FYI, the text of these bullets has not changed. I've just swapped the steps because of a reordering of statements in the code.